File: Constructorio_Net/src/models/RecommendationsResponseInner.java
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

/**
 * Constructor.io Recommendations Response Inner
 **/
namespace Constructorio_NET
{
    public class RecommendationsResponseInner {

        [JsonPropertyName("results")]
        public List<Result> results;

        [JsonPropertyName("total_num_results")]
        public Integer totalNumberOfResults;

        [JsonPropertyName("pod")]
        public ResultPod pod;

    }
}

File: Constructorio_NET/src/models/RecommendationsResponseInner.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

/**
 * Constructor.io Recommendations Response Inner
 */
namespace Constructorio_NET
{
    public class RecommendationsResponseInner {

        [JsonProperty("results")]
        public List<Result> Results { get; set; }

        [JsonProperty("total_num_results")]
        public Int16 TotalNumberOfResults { get; set; }

        [JsonProperty("pod")]
        public ResultPod Pod { get; set; }

    }
}